src/test/java/org/isf/admission/test/Tests.java

```diff
@@ -542,9 +474,560 @@ public void testIoDeletePatientPhoto() throws Exception {
 		assertThat(foundAdmission.getPatient().getPatientProfilePhoto().getPhoto()).isNull();
 	}
 
+	@Test
+	public void testIoLoadAdmittedPatientNotThere() throws Exception {
+		assertThat(admissionIoOperation.loadAdmittedPatient(-1)).isNull();
+	}
+
+	@Ignore
+	@Test
+	@Transactional
+	public void testIoLoadAdmittedPatient() throws Exception {
+		int id = _setupTestAdmission(false);
+		Admission admission = admissionIoOperation.getAdmission(id);
+		assertThat(admission).isNotNull();
+		assertThat(admissionIoOperation.loadAdmittedPatient(id)).isNotNull();
+	}
```

Comment:
Playing with this here: https://github.com/informatici/openhospital-core/pull/273
but tests is still failing: https://github.com/informatici/openhospital-core/pull/273/checks?check_run_id=1596783555